release-team/role-handbooks/release-signal/README.md

```diff
@@ -510,11 +510,12 @@ feel should be brought to the attention of the release team.
 
 ### A Tour of CI on the Kubernetes Project
 
-[A guide to CI on Kubernetes with Rob Kielty (1.20 CI Signal lead) and Dan Magnum (1.19 CI Signal lead)](https://www.youtube.com/watch?v=bttEcArAjUw):
+- [A guide to CI on Kubernetes with Rob Kielty (1.20 CI Signal lead) and Dan Magnum (1.19 CI Signal lead)](https://www.youtube.com/watch?v=bttEcArAjUw):
 Introduces the role of CI Signal members and walks through the tools and processes used for logging flaky/failing tests (as
 explained below).
+<br> Notes to follow along: http://bit.ly/k8s-ci-signal
 
-Notes to follow along: http://bit.ly/k8s-ci-signal
+- The [Release Signal Walkthroughs Playlist](https://youtube.com/playlist?list=PL69nYSiGNLP2Lzsjir9W7S8u0UsQeeW71&si=mp8zpMyheVR1eI1i) contains short videos explaining some essential aspects of Release Signal. 
 
```

Comment:
Thank you for working on this.
Really helpful resource, for everyone (folks in current RT & future applicants). :heart: